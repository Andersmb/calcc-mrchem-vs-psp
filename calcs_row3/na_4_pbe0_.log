Starting job 5688124 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 03:30:58 CEST 2022

Copying na_4_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying na_4_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688124      na_4_pbe0_        180                                             00:34:02      0:0 
5688124.bat+      batch         24        1   00:00:04          0   00:00:04   00:34:02      0:0 
5688124.ext+     extern        180        6   00:00:00          5   00:00:00   00:34:02      0:0 
5688124.0      mrchem.x        180       30   00:33:35         29   01:31:06   00:33:40      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688124                                                                          
5688124.bat+     19336K          0     19336K        0              0          0 
5688124.ext+          0          5          0        0              5          0 
5688124.0     15414448K         29 6557452834        0              8          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688124                                                                                               
5688124.bat+       19.44M               0         19.44M        0.07M                0          0.07M 
5688124.ext+        0.00M               5          0.00M            0                5              0 
5688124.0           7.32M              28          6.83M      161.09M                4         80.81M 

Job 5688124 completed at Fri May 20 04:05:01 CEST 2022
