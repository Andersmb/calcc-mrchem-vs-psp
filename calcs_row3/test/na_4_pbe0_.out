                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :       -1.789770        0.000000        1.960050
    1      Na            :        1.789770        0.000000       -1.960050
    2       F            :        2.146530        0.000000        1.960050
    3       F            :       -2.146530        0.000000       -1.960050
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              20
 Beta electrons          :                                              20
 Total electrons         :                                              40
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999990296260e-01
    1    1    a          :                              1.000000217493e+00
    2    1    a          :                              9.999998790427e-01
    3    1    a          :                              9.999999254305e-01
    4    1    a          :                              1.000001699475e+00
    5    1    a          :                              9.999971158647e-01
    6    1    a          :                              1.000000211484e+00
    7    1    a          :                              1.000000072586e+00
    8    1    a          :                              9.999928568736e-01
    9    1    a          :                              9.999922668077e-01
   10    1    a          :                              9.999995098547e-01
   11    1    a          :                              1.000001139766e+00
   12    1    a          :                              1.000000106269e+00
   13    1    a          :                              1.000000371410e+00
   14    1    a          :                              1.000000689683e+00
   15    1    a          :                              1.000000361480e+00
   16    1    a          :                              1.000000103110e+00
   17    1    a          :                              9.999998837646e-01
   18    1    a          :                              1.000001016302e+00
   19    1    a          :                              1.000000230002e+00
   20    1    b          :                              9.999992868423e-01
   21    1    b          :                              9.999997850288e-01
   22    1    b          :                              1.000000361492e+00
   23    1    b          :                              1.000002069717e+00
   24    1    b          :                              1.000001708041e+00
   25    1    b          :                              9.999997079978e-01
   26    1    b          :                              9.999998761252e-01
   27    1    b          :                              9.999996194673e-01
   28    1    b          :                              9.999996040003e-01
   29    1    b          :                              1.000002243945e+00
   30    1    b          :                              1.000000074599e+00
   31    1    b          :                              1.000001290463e+00
   32    1    b          :                              1.000000198580e+00
   33    1    b          :                              1.000000298390e+00
   34    1    b          :                              1.000000514408e+00
   35    1    b          :                              1.000000357660e+00
   36    1    b          :                              1.000000208029e+00
   37    1    b          :                              9.999997182725e-01
   38    1    b          :                              9.999999902729e-01
   39    1    b          :                              1.000000348863e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                520.393981541234
 E-N energy              :            (au)              -1533.288227971860
 Coulomb energy          :            (au)                404.969341292431
 Exchange energy         :            (au)                -12.184778886565
 X-C energy              :            (au)                -37.823318989261
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                137.328595617933
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -657.933003014022
 Nuclear energy          :            (au)                137.328595617933
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.206044073961e+02
                         :      (kcal/mol)             -3.266841952442e+05
                         :        (kJ/mol)             -1.366846673789e+06
                         :            (eV)             -1.416636698838e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.330313420269
    1    1    a          :            (au)                -38.327052626723
    2    1    a          :            (au)                -24.043982864701
    3    1    a          :            (au)                -24.029077324032
    4    1    a          :            (au)                 -2.226822800981
    5    1    a          :            (au)                 -2.222496761692
    6    1    a          :            (au)                 -1.138919497064
    7    1    a          :            (au)                 -1.138620886677
    8    1    a          :            (au)                 -1.136719041644
    9    1    a          :            (au)                 -1.134674146801
   10    1    a          :            (au)                 -1.133037507069
   11    1    a          :            (au)                 -1.131311641321
   12    1    a          :            (au)                 -0.796882743616
   13    1    a          :            (au)                 -0.774092637657
   14    1    a          :            (au)                 -0.098387750403
   15    1    a          :            (au)                 -0.093216835093
   16    1    a          :            (au)                 -0.086194835999
   17    1    a          :            (au)                 -0.080437110428
   18    1    a          :            (au)                 -0.065930492042
   19    1    a          :            (au)                 -0.059076735614
   20    1    b          :            (au)                -38.330331931127
   21    1    b          :            (au)                -38.327021399946
   22    1    b          :            (au)                -24.043976957554
   23    1    b          :            (au)                -24.029067230696
   24    1    b          :            (au)                 -2.226823827888
   25    1    b          :            (au)                 -2.222498308892
   26    1    b          :            (au)                 -1.138917507294
   27    1    b          :            (au)                 -1.138621725347
   28    1    b          :            (au)                 -1.136721012137
   29    1    b          :            (au)                 -1.134670984921
   30    1    b          :            (au)                 -1.133034932275
   31    1    b          :            (au)                 -1.131313594425
   32    1    b          :            (au)                 -0.796881075960
   33    1    b          :            (au)                 -0.774093693040
   34    1    b          :            (au)                 -0.098386518416
   35    1    b          :            (au)                 -0.093218485876
   36    1    b          :            (au)                 -0.086193047543
   37    1    b          :            (au)                 -0.080428799290
   38    1    b          :            (au)                 -0.065934877602
   39    1    b          :            (au)                 -0.059076060627
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -276.094459630684
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          6.324555e+00        -520.604407396089         -5.206044e+02
    1          7.363861e-01        -523.856437177656         -3.252030e+00
    2          4.319477e-01        -524.080950549781         -2.245134e-01
    3          3.684776e-01        -524.090120780048         -9.170230e-03
    4          1.834876e-01        -524.164222983602         -7.410220e-02
    5          3.107174e-02        -524.166477921552         -2.254938e-03
    6          2.274529e-02        -524.167431503562         -9.535820e-04
    7          6.655762e-03        -524.167522648329         -9.114477e-05
    8          4.438531e-03        -524.167537689782         -1.504145e-05
    9          1.922789e-03        -524.167543147930         -5.458148e-06
   10          9.825196e-04        -524.167544590069         -1.442139e-06
   11          4.797338e-04        -524.167544757763         -1.676938e-07
   12          2.389717e-04        -524.167544832060         -7.429730e-08
   13          7.944146e-05        -524.167544844484         -1.242438e-08
---------------------------------------------------------------------------
                      SCF converged in 13 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :       -1.789770        0.000000        1.960050
    1      Na            :        1.789770        0.000000       -1.960050
    2       F            :        2.146530        0.000000        1.960050
    3       F            :       -2.146530        0.000000       -1.960050
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                522.927100451134
 E-N energy              :            (au)              -1536.694572702359
 Coulomb energy          :            (au)                401.929656119463
 Exchange energy         :            (au)                -12.081432334374
 X-C energy              :            (au)                -37.576891996282
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                137.328595617933
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -661.496140462418
 Nuclear energy          :            (au)                137.328595617933
---------------------------------------------------------------------------
 Total energy            :            (au)             -5.241675448445e+02
                         :      (kcal/mol)             -3.289200977324e+05
                         :        (kJ/mol)             -1.376201689806e+06
                         :            (eV)             -1.426332489348e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.512492685032
    1    1    a          :            (au)                -38.512492139557
    2    1    a          :            (au)                -24.597401636261
    3    1    a          :            (au)                -24.597401420630
    4    1    a          :            (au)                 -2.217922777480
    5    1    a          :            (au)                 -2.217488836488
    6    1    a          :            (au)                 -1.137352275295
    7    1    a          :            (au)                 -1.133174978986
    8    1    a          :            (au)                 -1.132460152683
    9    1    a          :            (au)                 -1.131931722018
   10    1    a          :            (au)                 -1.128577663558
   11    1    a          :            (au)                 -1.127806628756
   12    1    a          :            (au)                 -1.019097235391
   13    1    a          :            (au)                 -1.016151235880
   14    1    a          :            (au)                 -0.306518807289
   15    1    a          :            (au)                 -0.302375756861
   16    1    a          :            (au)                 -0.296367613780
   17    1    a          :            (au)                 -0.293051850802
   18    1    a          :            (au)                 -0.287273648078
   19    1    a          :            (au)                 -0.284958720487
   20    1    b          :            (au)                -38.512492687206
   21    1    b          :            (au)                -38.512492132767
   22    1    b          :            (au)                -24.597401655492
   23    1    b          :            (au)                -24.597401430004
   24    1    b          :            (au)                 -2.217922785834
   25    1    b          :            (au)                 -2.217488827329
   26    1    b          :            (au)                 -1.137352277268
   27    1    b          :            (au)                 -1.133174988327
   28    1    b          :            (au)                 -1.132460151815
   29    1    b          :            (au)                 -1.131931718947
   30    1    b          :            (au)                 -1.128577662727
   31    1    b          :            (au)                 -1.127806633719
   32    1    b          :            (au)                 -1.019097256667
   33    1    b          :            (au)                 -1.016151249453
   34    1    b          :            (au)                 -0.306518842568
   35    1    b          :            (au)                 -0.302375770708
   36    1    b          :            (au)                 -0.296367626494
   37    1    b          :            (au)                 -0.293051879159
   38    1    b          :            (au)                 -0.287273675141
   39    1    b          :            (au)                 -0.284958758829
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -282.504595795766
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000001       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000001
                         :         (Debye)                        0.000002
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000001       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000001
                         :         (Debye)                        0.000002
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 33m 36s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
